cluster-sharding/src/main/scala/com/evolutiongaming/akkaeffect/cluster/sharding/ClusterSharding.scala

```diff
@@ -62,7 +67,7 @@ object ClusterSharding {
     val actorRefOf = ActorRefOf.fromActorRefFactory[F](actorSystem)
     val terminated = Terminated[F](actorRefOf)
 
-    def shardRegion(actorRef: => ActorRef) =
+    def createShardRegion(actorRef: => ActorRef) =
```

Comment:
isn't this the public API change?

shouldn't we deprecate with hit about new API the original?:
```scala
def shardRegion(actorRef: => ActorRef): Resource[F, ActorRef]
```

Reply:
It looks tricky but its not an API change, let me explain: I've added new API method
```
def shardRegion(typeName: TypeName): F[ActorRef]
```

and implement it on line #135. Unfortunately there was already a private method with same name and they clash in same scope. Thus I renamed private method to `createShardRegion`. Btw, could it be that MiMa by mistake took it as public API change? I have no idea how MiMa works and does it parses the code and builds ADT or just uses regex magic 🤔